view/summary.go

```diff
@@ -0,0 +1,55 @@
+package view
+
+import (
+	"fmt"
+	"math"
+)
+
+func (v *viewer) OutputSummary() {
+	if len(v.ctx.InProgressStats) == 0 {
+		return
+	}
+
+	if len(v.ctx.InProgressStats) == 1 {
+		stats := v.ctx.InProgressStats[0]
+
+		v.printer.Printf("\n--- %s ping statistics ---\n", v.ctx.Hostname)
+		v.printer.Printf("%d packets transmitted, %d received, %.2f%% packet loss, time %.0fms\n",
```

Comment:
For `time`, we need to account for the current `in-progress` measurement too. Let's do the following:
1. When you receive the response to POST, store that as a starting time.
2. Until the measurement is completed, print `current time - start time` (if the user aborts).
3. After the measurement completes, store the real time from the API and repeat the local timer thing for the new measurement.

So, at any point, the total time is: sum of all finished measurements as reported by the API + the `in-progress` measurement as measured locally.

Reply:
Maybe we can use measurement.CreatedAt - measurement.UpdatedAt?

Reply:
It would still be behind ~1 second. The closest estimate of duration should be the local time here.